work4.py

```diff
@@ -0,0 +1,9 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, 
+# что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+
+print('Введите общее количество журавликов')
+a = float(input())
+print('Петя и Серёжа сделали по ',  a/6, 'журавликов' , 'Катя сделала ' , a*2/3 , 'журавликов')
```

Comment:
В результате вашего решения могут получаться дробные числа, а такого быть не может. Невозможно сделать пол журавлика.
```suggestion
print('Петя и Серёжа сделали по ',  a / 6, 'журавликов' , 'Катя сделала ' , a * 2 / 3 , 'журавликов')
```
Так что это задание выполнено наполовину верно, наполовину неверно.